docs/src/techpapers/iris_xarray.rst

```diff
@@ -0,0 +1,130 @@
+.. include:: ../common_links.inc
+
+======================
+Iris ❤️ :term:`Xarray`
+======================
+
+There is a lot of overlap between Iris and :term:`Xarray`, but some important
+differences too. Below is a summary of the most important differences, so that
+you can be prepared, and to help you choose the best package for your use case.
+
+Overall Experience
```

Comment:
It'd be nice to point out that it can be easy to convert between the two.

Reply:
3277288c1475f98aa12c12e30f6c1107b96316a4